tools/anthosbm-ansible-module/roles/service-accounts/tasks/main.yml

```diff
@@ -12,115 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Enable Google Cloud APIs for Anthos
```

Comment:
The reason for keeping separate tasks for creating service account was to provide more control to the users. Many customers either create SAs separately or like to reuse them for multiple cluster. Please keep the original code. You can add a conditional task to create service account using bmctl create config.